CJSpohn/tic-tac-toe
# Feature/basic html css structure

Is this a fix or a feature?
Feature

What is the change/fix?
Basic HTML and CSS for our tic tac toe game

Where should the reviewer start?
In index.html and styles.css. We have no js content yet.

How should this be tested?
Open it up and make sure it looks similar to the comp. (In layout not theme)

Screenshots(if appropriate)
